packages/accounts-controller/src/AccountsController.ts

```diff
@@ -429,6 +435,29 @@ export class AccountsController extends BaseController<
     });
   }
 
+  /**
+   * Updates the metadata of the account with the given ID.
+   *
+   * @param accountId - The ID of the account to set the name for.
```

Comment:
```suggestion
   * @param accountId - The ID of the account for which the metadata will be updated.
```

Reply:
Dammnnn, went too fast with the copy paste, sorry! Updated!